Mathlib/Analysis/SpecialFunctions/Log/Base.lean

```diff
@@ -397,7 +397,47 @@ theorem logb_eq_zero : logb b x = 0 ↔ b = 0 ∨ b = 1 ∨ b = -1 ∨ x = 0 ∨
   simp_rw [logb, div_eq_zero_iff, log_eq_zero]
   tauto
 
--- TODO add other limits and continuous API lemmas analogous to those in Log.lean
+theorem tendsto_logb_nhdsWithin_zero (hb : 1 < b) :
+    Tendsto (logb b) (𝓝[≠] 0) atBot :=
+  tendsto_log_nhdsWithin_zero.atBot_div_const (log_pos hb)
+
+theorem tendsto_logb_nhdsWithin_zero_of_base_lt_one (hb₀ : 0 < b) (hb : b < 1) :
+    Tendsto (logb b) (𝓝[≠] 0) atTop :=
+  tendsto_log_nhdsWithin_zero.atBot_mul_const_of_neg (inv_lt_zero.2 (log_neg hb₀ hb))
+
+lemma tendsto_logb_nhdsWithin_zero_right (hb : 1 < b) : Tendsto (logb b) (𝓝[>] 0) atBot :=
+  tendsto_log_nhdsWithin_zero_right.atBot_div_const (log_pos hb)
+
+lemma tendsto_logb_nhdsWithin_zero_right_of_base_lt_one (hb₀ : 0 < b) (hb : b < 1) :
+    Tendsto (logb b) (𝓝[>] 0) atTop :=
+  tendsto_log_nhdsWithin_zero_right.atBot_mul_const_of_neg (inv_lt_zero.2 (log_neg hb₀ hb))
+
+theorem continuousOn_logb : ContinuousOn (logb b) {0}ᶜ := continuousOn_log.div_const _
+
+/-- The real logarithm base b is continuous as a function from nonzero reals. -/
+@[continuity]
```

Comment:
Do you really want this as a `continuity` lemma? I would suggest marking it `fun_prop` instead, but I don't see that as being particularly useful either. I simply suspect that this is never going to be useful as a continuity lemma. 
```suggestion
```

Reply:
I kept the `continuity` tag to mimic how it's done for `Real.log`. Would you prefer me to introduce a series of inconsistencies here?

Reply:
Oops, I just noticed you also suggested making the change there too! I'm testing to see what breaks if we remove it, and will check back here afterwards. I'm inclined to agree the tag in both places is useless, but let's see. That said, it could be nice as a discoverability tool - it's not like this is gonna slow down `fun_prop` a lot, but if someone happens to be in the case where they really are working with functions to/from `{ x : ℝ // 0 < x }`, then having `fun_prop` do this automatically does seem valuable.

Reply:
Seems to work! Let's go for it :)